lib/hound/helpers.ex

```diff
@@ -24,6 +24,8 @@ defmodule Hound.Helpers do
     quote do
       setup do
         Hound.start_session
+        on_exit fn-> Hound.end_session end
```

Comment:
@tuvistavie ah yes. Now I remember why I didn't do it long ago. Merging the PR was my mistake. I think I'll look into the tests soon to see how they passed this.